components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__phenotype.xml

```diff
@@ -482,6 +484,10 @@
   margin: .5em .75% 0em !important;
   padding-right: .75%;
 }
+.non-moving {
+  margin-top: 75px;
```

Comment:
Most of the UI is defined in `em` units, not in pixels. The actual height of the QS box is exactly 4em, plus 1.2 for the margin of the "Browse categories" header, set this margin to `5.2em` instead of `75px`. This way it will actually fit even if the brower uses larger fonts or a different DPI.